bot.py

```diff
@@ -66,6 +66,8 @@ async def help(interaction: nextcord.Interaction):
     response += "/choose \n- select a random item from a watchlist\n\n"
     response += "/status \n- update watch status of a movie or show in a watchlist\n\n"
     response += "/history \n- display all the watched movies in a watchlist\n\n"
+    response += "/filter_tags \n- display all the filtered movies based on tagsin a watchlist\n\n"
```

Comment:
Nice work adding this in for your functionality, however just a very minor typo that needs to be addressed.

```suggestion
    response += "/filter_tags \n- display all the filtered movies based on tags in a watchlist\n\n"
```